std/assembly/string.ts

```diff
@@ -168,7 +171,9 @@ export class String {
 
   @operator("<")
   private static __lt(left: String, right: String): bool {
-    if (left === null || right === null) return false;
+    if (left === right || left === null || right === null) {
+      return false;
+    }
```

Comment:
Just `if (left === right || left === null || right === null) return false;`, so it results in the following AST:

```
IfStatement
- Expression
- ReturnStatement
```

instead of

```
IfStatement
- Expression
- BlockStatement
  - ReturnStatement
```

Reply:
I refactor this. But I can't see any changes even in `untouched` wast. Is it normal?

Reply:
Yeah, it does compile to the same thing, but the parser has to allocate one less AST node. My though was that, since stdlib is always included in compilation and fully parsed (unlike other compilers that might link it in), that it would accumulate.

Reply:
ok, got it